### DISCUSSION TITLE
After table load its dump all data in query string for current url.

### DISCUSSION THREAD

**Discussion by vimal-360** [rappasoft/laravel-livewire-tables#1151] [Q&A]
Using version 1.x

Quick Help for Production App:

When the table load the first time after my current URL changes and its looks like this.

![Screenshot 2023-04-10 at 12 06 18 PM](https://user-images.githubusercontent.com/95009456/230842913-540c3e34-0e2d-4018-8f7c-084623486141.png)

**Comment by lrljoe** [comment]
Not too familiar with 1.x I'm afraid!

Can I ask for what is preventing you from migrating to 2.x?  If there's something missing then I can look to get it into a future version.

**Reply by vimal-360** [reply]
There are too many tables in my app and it is in production. The new version has many breaking changes. it take time :/

**Reply by lrljoe** [reply]
Okay, v1.x hasn't been updated very recently, so I'm assuming this is unrelated to any recent changes?

If its a new issue with the Component, then can you please share your Laravel, Livewire, AlpineJS, and Tailwind/BootStrap versions please :)

**Reply by vimal-360** [reply]
Bootstrap 5
Laravel 9.52.5
Livewire 2.12.3
Alpine js ^3.10.2

**Reply by lrljoe** [reply]
What does your queryString look like:

```
    protected $queryString = [ 
         'filters' => ['except' => null], 
         'sorts' => ['except' => null], 
     ]; 
 ```
